lib/_internal/_curry2more/index.js

```diff
@@ -0,0 +1,10 @@
+export default function curry2more(f) {
+  return function curried() {
```

Comment:
I see, you don't call `curried` function anywhere, could you use anonymous function? It would reduce method's size

Reply:
@BabkinAleksandr it's for debugging sake